UltiSnips/nix.snippets

```diff
@@ -0,0 +1,51 @@
+snippet sha256 "sha256"
+sha256 = "0000000000000000000000000000000000000000000000000000";
+endsnippet
+
+snippet fetchFromGitHub "fetchFromGitHub"
+fetchFromGitHub {
+  owner = "${1}";
+  repo = "${2}";
+  rev = "${3:v\${version}}";
+  sha256 = "0000000000000000000000000000000000000000000000000000";
+}
+endsnippet
+
+snippet fetchurl "fetchurl"
+fetchurl {
+  url = "${1}";
+  sha256 = "0000000000000000000000000000000000000000000000000000";
```

Comment:
I guess these days we may want to use `hash = "sha256-AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA=";` here instead.